assets/js/frontend/cart.js

```diff
@@ -71,7 +71,7 @@ jQuery( function( $ ) {
 
 		// No form, cannot do this.
 		if ( $( '.woocommerce-cart-form' ).length === 0 ) {
-			window.location.href = window.location.href;
+			window.location.refresh();
```

Comment:
I think we might want to use `window.location.reload()` here as `refresh()` is not a function.

Reply:
Good catch, thanks. It's been fixed